mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -250,13 +288,17 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
         */
         app.commands.addCommand(COMMAND_MITO_AI_APPLY_LATEST_CODE, {
             execute: () => {
-                acceptAICode()
+                const newChatHistoryManager = getDuplicateChatHistoryManager()
+                const codeCellID = newChatHistoryManager.getCodeCellIDOfMostRecentAIMessage() || ''
+                acceptAICode(codeCellID)
             }
         })
 
         app.commands.addCommand(COMMAND_MITO_AI_REJECT_LATEST_CODE, {
             execute: () => {
-                rejectAICode()
+                const newChatHistoryManager = getDuplicateChatHistoryManager()
+                const codeCellID = newChatHistoryManager.getCodeCellIDOfMostRecentAIMessage() || ''
+                rejectAICode(codeCellID)
```

Comment:
This is good for now. I wonder if it would be better if the `rejectAICode` and `acceptAICode` functions lived in the chatHistoryManager instead of here?